rust/actyx/ax/src/cmd/mod.rs

```diff
@@ -59,19 +59,19 @@ impl ConsoleOpt {
     }
 }
 
-pub(crate) mod consts {
-    use prettytable::format::{FormatBuilder, LinePosition, LineSeparator, TableFormat};
-    lazy_static::lazy_static! {
-        pub static ref TABLE_FORMAT: TableFormat = FormatBuilder::new()
-            .column_separator('│')
-            .borders('│')
-            .separators(&[LinePosition::Top], LineSeparator::new('─', '┬', '┌', '┐'))
-            .separators(&[LinePosition::Title], LineSeparator::new('─', '┼', '├', '┤'))
-            .separators(&[LinePosition::Bottom], LineSeparator::new('─', '┴', '└', '┘'))
-            .padding(1, 1)
-            .build();
-    }
-}
+// pub(crate) mod consts {
```

Comment:
Is this kept for, in the future, referring back to old table style?

Reply:
Nope, because I forgot to delete. Nice catch